### ISSUE TITLE
Is there any possibility of an ARM Image for those of us using Oracle TestContainers on MacOS?

### ISSUE THREAD

**Issue by nigel-trousdale** [gvenzl/oci-oracle-free#46]
https://www.oracle.com/database/technologies/oracle19c-linux-arm64-downloads.html

**Comment by nigel-trousdale** [comment]
Making the wild assumption that it could be compatible with your oracle-xe container.

**Comment by gvenzl** [comment]
Hi @nigel-trousdale,

Thanks a lot for your interest in these images!

Unfortunately, not just yet.
There is currently no Oracle Database 23c Free ARM port available to build a container image from.
However, Oracle has stated that an ARM port for Oracle Database 23c Free is coming later this year.
Once it is out, I will be on it to provide arm-based images as well.

**Comment by nigel-trousdale** [comment]
Thanks @gvenzl . We are making great use of your images with TestContainers for our integration testing, but many of the team are now on Intel Macs, which causes quite some issues. I haven't found a way to use different docker contexts for different test containers otherwise that might be an option.

**Comment by hfhbd** [comment]
Thanks for sharing its planned later this year!

We also switched to Apple Silicon Macs and use Java Testcontainers. While running a amd64 container _should_ work with colima, setting up the correct environment is quite complex and it is often broken due some changes in colima/docker and macOS itself, when using macOS Virtualization. A solution out of the box using arm64 images without colima sounds perfect. 

[Last year, the developer release of 23c was released in April](https://blogs.oracle.com/database/post/oracle-database-23c-free). A similar time schedule would be nice to give you/Oracle faster feedback :)
